account_banking_mandate/views/res_partner_bank_view.xml

```diff
@@ -10,11 +10,12 @@
         <field name="inherit_id" ref="base.view_partner_bank_form" />
         <field name="arch" type="xml">
             <xpath expr="//form/sheet/group" position="inside">
-                <group name="mandates" string="Direct Debit Mandates">
+                <group name="mandates" string="Direct Debit Mandates" col="4">
```

Comment:
You shouldn't change the `col`. If the problem is that the width is not completed, please check on the migration guide how it should be done:

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

and for these things, always put a screenshot of the before and after for easing reviews.